src/uwtools/utils/tasks.py

```diff
@@ -35,14 +43,26 @@ def executable(program: Union[Path, str]):
 
 
 @external
-def existing(path: Path):
+def existing(path: Union[Path, str]):
```

Comment:
This task function has been updated to check the existence of both local filesystem items and remote network objects. The latter will be represented as `str`s, so the function signature is updated.

For local paths, remove a `file://` prefix if present, then execute the same logic as before.

For HTTP paths, let `requests` do a `HEAD` check for existence.

For other network schemes, raise an exception.